.github/workflows/prepare-release.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         with:
           branch: release/${{ steps.update-changelog.outputs.new-version }}
           base: master
-          title: [Release ${{ steps.update-changelog.outputs.new-version }}]
+          title: '[Release ${{ steps.update-changelog.outputs.new-version }}]'
```

Comment:
This was introduced in https://github.com/realm/realm-dart/pull/849 and is invalid yaml - we need to escape the array elements or wrap the entire thing in a string.